addons/account/models/account_report.py

```diff
@@ -361,14 +362,17 @@ def _inverse_domain_formula(self):
     def _inverse_aggregation_formula(self):
         self._create_report_expression(engine='aggregation')
 
+    def _inverse_aggregation_tax_formula(self):
+        self._create_report_expression(engine='aggregation', label='taxbalance')
```

Comment:
What ? :thinking:  Not sure you understood what you were doing ^^

```suggestion
        self._create_report_expression(engine='tax_tags', label='balance')
```

Just like for other shortcuts. Someone wanting to use another label will have to use the full syntax